### PR TITLE
Update beta to GNOME 40 runtime

### PR DESCRIPTION
**Pull request by linkmauve**
This is just a merge of master after #74.

### CODE REVIEW

**Comment by hfiguiere** [comment]
LOL. I had the branch locally just that it wasn't done building last night... and hadn't pushed it yet.

File: org.inkscape.Inkscape.json [hunk 1]
```diff
@@ -425,6 +425,10 @@
                 "-DBUILD_TESTING=OFF",
                 "-DWITH_INTERNAL_2GEOM=ON"
             ],
+            "build-options": {
+                "cppflags": "-DGLIB_VERSION_MIN_REQUIRED=GLIB_VERSION_2_66",
```
Comment (hfiguiere) [inline org.inkscape.Inkscape.json:429]: It's not important, and I let it through on stable, is that this `cppflags` in that case is ignored, therefor unnecessary.
(it works for other modules though)

**Comment by hfiguiere** [comment]
LGTM but I would just have cherry-picked the patches

File: org.inkscape.Inkscape.json [hunk 2]
```diff
@@ -10,10 +10,11 @@
     "desktop-file-name-prefix": "(Beta) ",
     "finish-args": [
         "--share=ipc",
-        "--socket=x11",
+        "--socket=fallback-x11",
         "--socket=wayland",
         "--filesystem=host",
         "--filesystem=xdg-run/gvfs",
+        "--filesystem=xdg-run/gvfsd",
         "--talk-name=org.gtk.vfs",
```
Comment (hfiguiere) [inline org.inkscape.Inkscape.json:18]: Also related to the change above, this line should be removed. See https://github.com/flathub/flathub/issues/2180#issuecomment-811984901

(I don't have that change locally either)

Comment (linkmauve) [reply org.inkscape.Inkscape.json:18]: Is the `org.gtk.vfs.*` line still needed though?

Comment (hfiguiere) [reply org.inkscape.Inkscape.json]: yes. Just that `org.gtk.vfs` doesn't exist and it seems to be part of the cut & paste cargo cult spread through out log of packages. I think I'm the guilty party here.

**Comment by hfiguiere** [comment]
LGTM

**Comment by linkmauve** [comment]
Err, I misread your comment, sorry. >_<

**Comment by linkmauve** [comment]
There, it’s back to the version you reviewed, you can merge it. :)

**Pull request merged**